### PR TITLE
PT-6834: The ability to set a priority for each sitemap for each type

### PR DESCRIPTION
**Pull request by VertyR**
#PT-6834

## Description
It is necessary to be able to set the fields "update frequency" and "priority" to the default values.

### CODE REVIEW

File: src/VirtoCommerce.SitemapsModule.Core/ModuleConstants.cs [hunk 1]
```diff
@@ -148,8 +148,45 @@ public static IEnumerable<SettingDescriptor> AllSettings
...
+                };
+
+                public static readonly SettingDescriptor BlogPagePriority = new SettingDescriptor
+                {
+                    Name = "Sitemap.BlogPagePriority",
+                    GroupName = "Sitemap|Sitemap Blog Links",
+                    ValueType = SettingValueType.Decimal
+                };
+
+                public static IEnumerable<SettingDescriptor> AllSettings
```
Comment (Vectorfield4) [inline src/VirtoCommerce.SitemapsModule.Core/ModuleConstants.cs:177]: Please, disable warning S3218

File: src/VirtoCommerce.SitemapsModule.Data/Services/SitemapItemRecordProviders/CatalogSitemapItemRecordProvider.cs [hunk 2]
```diff
@@ -135,21 +132,36 @@ protected virtual async Task LoadProductsSitemapItemRecordsAsync(Store store, Si
             }
         }
 
-        private SitemapItemOptions GetProductOptions()
+        private SitemapItemOptions GetStoreOptions(Store store)
```
Comment (Vectorfield4) [inline src/VirtoCommerce.SitemapsModule.Data/Services/SitemapItemRecordProviders/CatalogSitemapItemRecordProvider.cs:135]:
```suggestion
        private static SitemapItemOptions GetStoreOptions(Store store)
```

**Review by Vectorfield4 — APPROVED** [review]

File: src/VirtoCommerce.SitemapsModule.Core/ModuleConstants.cs (hunk 1 above)
Comment (VertyR) [reply src/VirtoCommerce.SitemapsModule.Core/ModuleConstants.cs:179]: Fixed

File: src/VirtoCommerce.SitemapsModule.Data/Services/SitemapItemRecordProviders/CatalogSitemapItemRecordProvider.cs (hunk 2 above)
Comment (VertyR) [reply src/VirtoCommerce.SitemapsModule.Data/Services/SitemapItemRecordProviders/CatalogSitemapItemRecordProvider.cs]: Fixed

**Review by Vectorfield4 — DISMISSED** [review]

**Review by Vectorfield4 — APPROVED** [review]

**Pull request merged**